user_tools/src/spark_rapids_tools/utils/util.py

```diff
@@ -323,3 +323,17 @@ def bytes_to_human_readable(cls, num_bytes: int) -> str:
             num_bytes /= 1024.0
             i += 1
         return f'{num_bytes:.2f} {size_units[i]}'
+
+    @classmethod
+    def get_hadoop_conf_dir(cls) -> Optional[str]:
+        """
+        Get Hadoop's configuration directory from the environment variables.
+        If not defined, return $HADOOP_HOME/etc/hadoop if HADOOP_HOME is defined.
+        Otherwise, returns None.
+        """
+        hadoop_conf_dir = Utils.get_sys_env_var('HADOOP_CONF_DIR')
+        if hadoop_conf_dir is None:
+            hadoop_home = Utils.get_sys_env_var('HADOOP_HOME')
+            if hadoop_home is not None:
+                hadoop_conf_dir = os.path.join(hadoop_home, 'etc', 'hadoop')
```

Comment:
why are we doing /etc/hadoop ?  is this like EMR or something. Normal HADOOP has a default of $HADOOP_HOME/conf.   If it is some csp specifica thing, I would rather us check if HADOOP_CONF_DIR is set, check is exists, then check HADOOP_HOME/conf exists, then finally check HADOOP_HOME/etc/hadoop... Also document where /etc/hadoop is used.

Reply:
We are doing `$HADOOP_HOME/etc/hadoop`
I remember that there was different setting across hadoop releases (2.x vs 3.x) 
When I build hadoop `-Pdist` it generated an `etc/hadoop` inside the generated dist folder.
Let me modify the implementation to follow the same steps.

Reply:
https://github.com/apache/hadoop/blob/68fcd7234ca271b99cb26ba2759d873ade311c13/hadoop-common-project/hadoop-common/src/main/conf/hadoop-env.sh#L72

Reply:
ok grepping code though:
hadoop-common-project/hadoop-common/src/main/bin/workers.sh:#   HADOOP_CONF_DIR  Alternate conf dir. Default is ${HADOOP_HOME}/conf.

So lets just do both like I mention